app/main.py

```diff
@@ -4,0 +1,30 @@
+import json
+from typing import Callable
+
+from app.shop import Shop
+from app.customer import Customer
+
+
+def get_data(info_key: str, creation_func: Callable) -> list:
+    with open("app/config.json", "r") as source:
+        data = json.load(source)
+    return [creation_func(info) for info in data[info_key]]
```

Comment:
Ensure that the keys 'customers' and 'shops' are present in the JSON data to avoid KeyError. Consider using the `get` method with default values or handling exceptions.